tests/Button/Button.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import { shallow } from "enzyme";
+import  {Button} from "../../src/components/Button"
+
+describe("Tests for button component",() =>{
```

Comment:
Looks like there are some styling issues here. Hopefully, once we land #11 this will be automated.